Doc/library/py_compile.rst

```diff
@@ -22,6 +22,36 @@ shared use, especially if some of the users may not have permission to write the
 byte-code cache files in the directory containing the source code.
 
 
+Command-line usage
```

Comment:
It would be nice to use the wording in tarfile CLI docs: https://docs.python.org/3/library/tarfile.html#command-line-interface

Reply:
done